blockly/generators/propc.js

```diff
@@ -337,7 +340,8 @@ Blockly.propc.finish = function (code) {
             definitions[def] = definitions[def].replace(/\{\{\$var_type_.*?\}\}/ig, "int").replace(/\{\{\$var_length_.*?\}\}/ig, '');
         }
 
-        if (def.indexOf(cCode) === -1) {  // exclude custom code blocks from these modifiers
+            console.log(def_names[def]);
```

Comment:
This appears to be debugging code. Does it have another purpose?

Reply:
No, it can be removed.